src/no-flashcards.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function NoFlashcard(props) {
+  return (
+    <div className="container-fluid d-flex justify-content-center">
+      <div>
+        <h3>You have no flash cards</h3>
+        <div className="container-fluid d-flex justify-content-center">
+          <a href="#create"><button onClick={props.takeToForm} type="button" className="btn btn-dark">Create flash cards</button></a>
```

Comment:
If your `href` is set to the correct hash, the `window` will dispatch a `'hashchange'` event for you, so you don't need a click listener.

You can style `<a>` elements to look like buttons with Bootstrap (`className="btn btn-dark"`).